docs/core/diagnostics/diagnostics-in-containers.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Collect diagnostics in containers
+description: In this article, you'll learn how .NET Core diagnostics tools can be used in Docker containers.
+ms.date: 09/01/2020
+---
+
+# Collect diagnostics in containers
+
+**This article applies to: ✔️** .NET Core 2.1 SDK and later versions
```

Comment:
```suggestion
**This article applies to: ✔️** .NET Core 3.0 SDK and later versions
```

While the tools can be installed using a 2.1 SDK, they aren't able to work with a 2.1 application.  I saw you called this out later, but the docs page will have 2.1 specifically called out in bold font at the top of the page and users might miss your caveat statement.  I'm not 100% on what the requirements are for this statement in the docs though, so I might be wrong in suggesting this change.  @adegeo can you chime in here?

Reply:
I would go with 3.1 as 3.0 is out of support.

```suggestion
**This article applies to: ✔️** .NET Core 3.1 SDK and later versions
```

Reply:
Yea, I wasn't sure what to put for version here because some of the tools I mention (createdump) work on 2.1, others (dotnet-dump, dotnet-trace, dotnet-counters) work with 3.0, and one (dotnet-gcdump) requires 3.1. My solution was to use the earliest version and try to call out what requires a later one.

@adegeo - do we have a usual practice when writing conceptual docs like this that apply differently to different versions?

Reply:
I would use something similar to the "this article applies to" syntax but reword it for "this command" or "this tool" in the applicable section.

```markdown
## Using `PerfCollect` in a container

**This tool applies to: ✔️** .NET Core 3.1 SDK and later versions.

## Using `CreateDump` in a container

**This tool applies to: ✔️** .NET Core 2.1 SDK and later versions.

blah blah blah blah text text paragraph paragraph ...

blah blah blah blah text text paragraph paragraph ...
```

Reply:
Makes sense. I've removed the "This article applies to..." and added more specific "This tool applies to..." annotations for the different tools.